tests/tests_upgrade.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# This tests that the database upgraded by PUM is consistent to the database
+# initialized by qwat_init.sh
+#It retrieves the
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+INIT_DB=1
+
+while [[ $# > 0 ]]; do
+key="$1"
+case $key in
+    -h|--help)
```

Comment:
bash builtin long option names with the double-dash prefix. This is an extension from GNU getopt (on macos and bsd-like you have to install gnu-getopt).

More details: https://github.com/qwat/qwat-data-model/pull/228
and from Denis https://stackoverflow.com/a/37485578 :)

Reply:
I copied this from `tests.sh`, thus I suggest to keep it like that for consistency

(to be honest, no idea what this is about :-D)